### DISCUSSION TITLE
How to set custom styles on openConnectModal() modal?

### DISCUSSION THREAD

**Discussion by jasonyjong** [rainbow-me/rainbowkit#880] [Q&A]
Currently, I have `'pointer-events: none'` set on `body`, but since there is no `pointer-events` on the `openConnectModal`, it inherits from the `pointer-events: none` on the `body`. So whenever this modal opens, I am unable to click on the buttons in this modal.

Is there a way for me to set a custom style `'pointer-events: all'` for my connect wallet modal?

**Comment by adrianotadao** [comment]
@jasonyjong I think the only way of doing it is by overriding rainbowkit styles creating your own file and use `!important` everywhere xD
